src/main/java/uk/gov/hmcts/darts/event/service/impl/EventServiceImpl.java

```diff
@@ -34,4 +42,19 @@ public EventEntity getEventByEveId(Integer eveId) {
     public EventEntity saveEvent(EventEntity eventEntity) {
         return eventRepository.save(eventEntity);
     }
+
+    @Override
+    public Set<EventEntity> getAllCourtCaseEventVersions(CourtCaseEntity courtCase) {
+        Set<EventEntity> allEvents = new HashSet<>();
+        List<EventEntity> eventsFromCaseId = eventLinkedCaseRepository
```

Comment:
Maybe worth adding a comment to this method explaining that the way this table is populated differs between legacy and modernised, hence the reason for 2 different queries to the eventLinkedCaseRepository.

Reply:
Added